README.md

```diff
@@ -169,17 +170,45 @@ include:
 
 ---
 
+### WebTAK Plugin Test Coverage Pipeline
+
+The standard NPM pipeline is the simplest way to get up and running quickly. It provides a full pipeline configuration
+that will install, build, test, and provide test results and coverage reports on both merge requests and gitlab pages.
+
+#### Requirements
+
+The pipeline has this repo's `NpmJestCoveragePipeline` as an included dependency. It is necessary to follow the
+requirements from that pipeline in order to use this one.
+
+#### Customization
+
+| Variable      | Description                                                                           |
+|---------------|---------------------------------------------------------------------------------------|
+| TAK_EMAIL     | A valid TAK.gov email account with access to WebTAK artifacts on the TAK Artifactory. |
+| TAK_API_TOKEN | An API token generated using the same email as above.                                 |
+
+#### Reference URL
+
+```
+include:
+    - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/release/1.2/lib/gitlab/ci/templates/pipeline/WebtakTestCoverage.yml
+
+```
+
+---
+
```

Comment:
Calling out this block. This is the only relevant change to the README. The rest are just table reformatting to remove weak warnings in my IDE and satisfy my anal-retentiveness.